gui.py

```diff
@@ -0,0 +1,99 @@
+from conversion_manager import ConversionManager
+from path_helper import PathHelper
+from PyQt6.QtCore import QSize
+from PyQt6.QtGui import QIcon
+from PyQt6.QtWidgets import QApplication, QMainWindow, QPushButton, QVBoxLayout, QHBoxLayout, QLabel, \
+    QLineEdit, QWidget, QFileDialog, QMessageBox
+
+
+class MainWindow(QMainWindow):
+    def __init__(self, conversion_manager: ConversionManager, input_path: str, output_path: str) -> None:
+        super().__init__()
+        self.setMinimumSize(QSize(450, 150))
+
+        self.conversion_manager: ConversionManager = conversion_manager
+
+        self.setWindowTitle("Аршин-конвертер")
+        self.layout = QVBoxLayout()
+        input_layout = QHBoxLayout()
+        output_layout = QHBoxLayout()
+
+        input_label = QLabel('Выбрать файл')
```

Comment:
"Input" could be ambiguous in this case as test input controls are also often called "input".